packages/manager/core/order/package.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Because the order constants and functions are duplicated in multiple packages

And it's probably a good idea to centralize order functions like polling the order list because it will probably be used on multiple projects